scraper/entrypoint.sh

```diff
@@ -3,9 +3,9 @@
 echo "Running scraper"
 # this command will run the scraper, but it will not keep the container running
 # for development purposes, we don't need to run the scraper every time we run the container
-# python /opt/jobhub-scraper/main.py
+python /opt/jobhub-scraper/main.py
 
 
 # Then keep the container running
-tail -f /dev/null
-echo "Scraper container is up" 
\ No newline at end of file
+# tail -f /dev/null
```

Comment:
can remove this? unused

Reply:
no? just in case someone wants to connect to container and do some development, this  keeps the container running so we can attach to it